packages/twenty-server/.env.example

```diff
@@ -74,4 +74,8 @@ FRONTEND_URL=http://localhost:3001
 # SSL_CERT_PATH="./certs/your-cert.crt"
 # CLOUDFLARE_API_KEY=
 # CLOUDFLARE_ZONE_ID=
-# CLOUDFLARE_WEBHOOK_SECRET=
\ No newline at end of file
+# CLOUDFLARE_WEBHOOK_SECRET=
```

Comment:
Can we add a clickhouse image in Github action server-ci (can be done in a separate PR)? This would allow for basic test coverage, probably the sooner the better before we accumulate too much debt